distribution/lib/Standard/Visualization/0.0.0-dev/src/Widgets.enso

```diff
@@ -1,15 +1,26 @@
 from Standard.Base import all
+from Standard.Base.Meta import Instrumentor
 
 ## PRIVATE
    Basic preprocessor for widgets metadata visualization.
 
    Returns full annotation data for all requested arguments.
 get_widget_json : Any -> Any -> Vector Text -> Text
 get_widget_json value call_name argument_names =
+    other_args = argument_names.last . parse_json
```

Comment:
#### Incompatible API change 

The last string in the `argument_names` array is treated differently than the previous ones. We could do the change compatibly by adding new `get_widget_json_with_argument_infos` and changing IDE to call the new method, if available. Old IDEs would continue to call the unmodified version of the `get_widget_json` method. Opinions @jdunkerley, @farmaazon?

Reply:
I'm ok with calling a new method. Also, I'd prefer having a completely new API instead of making magic "last argument name" which is not an argument name.

Reply:
Compatibility of old IDE with new engine achieved by 47c7afb